modules/payloads/singles/cmd/unix/bind_zsh.rb

```diff
@@ -9,7 +9,7 @@
 
 module MetasploitModule
 
-  CachedSize = 112
+  CachedSize = 99
```

Comment:
Cool, thank you so much for your code review~
I am not sure what does the variable `CachedSize` mean, could you please elaborate that?

Reply:
It's just to speed up framework loads. See #4894. I ran `tools/modules/update_payload_cached_sizes.rb`.

Reply:
Got it. thank you :+1: